javav2/example_code/iotsitewise/src/main/java/com/example/iotsitewise/HelloSitewise.java

```diff
@@ -0,0 +1,66 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.example.iotsitewise;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.iotsitewise.IoTSiteWiseAsyncClient;
+import software.amazon.awssdk.services.iotsitewise.model.AssetSummary;
+import software.amazon.awssdk.services.iotsitewise.model.ListAssetsRequest;
+import software.amazon.awssdk.services.iotsitewise.paginators.ListAssetsPublisher;
+
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+
+// snippet-start:[iotsitewise.hello.main]
+public class HelloSitewise {
+    private static final Logger logger = LoggerFactory.getLogger(HelloSitewise.class);
+    public static void main(String[] args) {
+        final String usage = """
+            Usage:
+               <assetModelId>
+
```

Comment:
The fact that this relies on the user already having an asset model, makes it seem inappropriate for a Hello World example. Can you create the model for them. Or find a different action? maybe createAssetModel, but tell the user how to manually delete it in the console.

Reply:
I wil update to use a service operation that does not require any input values.